lib/usb_hid_host/usb_hid_keyboard.cpp

```diff
@@ -0,0 +1,211 @@
+#include <Arduino.h>
+#include <esp_log.h>
+#include "usb_hid_keyboard.h"
+
+#include "usb_hid_host.h"
+
+
+#include "hid_usage_keyboard.h"
+
+static const char* TAG = "usb-hid-keyboard";
+
+/**
+ * @brief Scancode to ascii table
+ */
+const uint8_t keycode2ascii[57][2] = {
```

Comment:
In general: I think keycode to ascii lookups are only good for visualising.
I think we should transfer keycodes 1:1 into the BLE/I2C reports.